bedrock/firefox/views.py

```diff
@@ -655,3 +666,11 @@ def get_template_names(self):
             template = 'firefox/feedback/unhappy.html'
 
         return [template]
+
+
+def android(request):
+    # check for variant in querystring for send-to-device testing
+    variant = request.GET.get('v', '')
```

Comment:
Is this a possible xss vector seeing as it gets embedded in the template? Would it be better to limit the possible values to either `a`, `b`, or `c`?